### ISSUE TITLE
predict 不能进行预测，求大佬讲解一下

### ISSUE THREAD

**Issue by ForgetThatNight** [saiwaiyanyu/bi-lstm-crf-ner-tf2.0#14]
这是输出的日志记录，代码没有动过
`input:我是范冰冰
dataset: [[ 131 2067 3239 2234 2234]]
logits
[[[-0.72535753  4.161972   -2.9507422  -3.1868236  -3.2397275
   -3.269389   -3.0774546  -3.2384624  -3.2812405  -2.9897215
   -3.1217413  -3.6321018  -3.1315987  -3.0792222 ]
  [-2.8050935   4.9421215  -2.3537152  -2.844575   -2.8227744
   -2.926732   -2.4319122  -2.8924425  -2.7267509  -2.4868493
   -2.7142     -3.325798   -2.5980093  -2.7135816 ]
  [-0.40876862  1.085415   -1.0298109  -1.1841301  -1.3309526
   -1.5018826  -1.0938866  -1.1450801  -1.3572048  -1.1883298
   -1.3080059  -1.4665012  -1.1049844  -1.3270234 ]
  [ 0.03211104  0.3558294  -0.6737744  -0.6955142  -0.8401693
   -0.9766958  -0.69395465 -0.74898857 -0.8396182  -0.77023965
   -0.86173874 -0.88923264 -0.68174887 -0.8035339 ]
  [ 0.13178386 -0.05146006 -0.43199152 -0.3135421  -0.44383863
   -0.5618567  -0.37870246 -0.36361524 -0.4879145  -0.49257016
   -0.50506824 -0.50199145 -0.37561214 -0.4440806 ]]]
text_lens
[5]
viterbi_path,va: [1, 1, 1, 1, 1] 15.173229
paths: [[1, 1, 1, 1, 1]]
[id2tag[id] for id in paths[0]]: ['O', 'O', 'O', 'O', 'O']
entities_result: []
json_result: []`

**Comment by ForgetThatNight** [comment]
> 
`
input:我是范冰冰
dataset: [[ 131 2067 3239 2234 2234]]

logits
[[[-0.72535753  4.161972   -2.9507422  -3.1868236  -3.2397275
   -3.269389   -3.0774546  -3.2384624  -3.2812405  -2.9897215
   -3.1217413  -3.6321018  -3.1315987  -3.0792222 ]
  [-2.8050935   4.9421215  -2.3537152  -2.844575   -2.8227744
   -2.926732   -2.4319122  -2.8924425  -2.7267509  -2.4868493
   -2.7142     -3.325798   -2.5980093  -2.7135816 ]
  [-0.40876862  1.085415   -1.0298109  -1.1841301  -1.3309526
   -1.5018826  -1.0938866  -1.1450801  -1.3572048  -1.1883298
   -1.3080059  -1.4665012  -1.1049844  -1.3270234 ]
  [ 0.03211104  0.3558294  -0.6737744  -0.6955142  -0.8401693
   -0.9766958  -0.69395465 -0.74898857 -0.8396182  -0.77023965
   -0.86173874 -0.88923264 -0.68174887 -0.8035339 ]
  [ 0.13178386 -0.05146006 -0.43199152 -0.3135421  -0.44383863
   -0.5618567  -0.37870246 -0.36361524 -0.4879145  -0.49257016
   -0.50506824 -0.50199145 -0.37561214 -0.4440806 ]]]

text_lens
[5]

viterbi_path,va: [1, 1, 1, 1, 1] 15.173229

paths: [[1, 1, 1, 1, 1]]

[id2tag[id] for id in paths[0]]: ['O', 'O', 'O', 'O', 'O']

entities_result: []

json_result: []
`
